### ISSUE TITLE
a way drag and reorder items only after or before a certain index

### ISSUE THREAD

**Issue by thorizer** [computerjazz/react-native-draggable-flatlist#340]
**Is your feature request related to a problem? Please describe. **
For example, a to do list or items that requires timeline/timestamps or have a status (done - to be done - in progress) .
I don't want the user to reorder the items in a non-chorological way. 
Moreover, the item shouldn't be draggable if its status is done for example.

**Describe the solution you'd like**
A way to drag and reorder items only before and after a certain index or based on a condition. 

**Describe alternatives you've considered**
`<item Drag={  before_Limit, after_Limit,  Optionnal_Condition)  />`

**Comment by computerjazz** [comment]
this is out of scope for this project.
As a workaround, you could render your before/after items outside of the draggable flatlist. For items that should not be draggable, don't call `drag()` on press.
